config/plugins.js

```diff
@@ -64,5 +64,22 @@ module.exports = ({ env }) => ({
         name: "strapi-prometheus",
       },
     }
-  }
-});
\ No newline at end of file
+  },
+  email: {
+    config: {
+      provider: 'nodemailer',
+      providerOptions: {
+        host: 'smtp.gmail.com',
+        port: 465,
+        auth: {
+          user: "email@gmail.com", // edit here
+          pass: "zzzz zzzz zzzz zzzz", // edit here
+        },
+      },
+      settings: {
+        defaultFrom: 'hello@example.com',
```

Comment:
@chungthuang @Enjoy2Live @annaisaacks @leeah-d is there an email address we want to use as the "from" address by default for dev launchers email interactions? We want this to be something professional because users may encounter emails during their onboarding and ongoing contributions at dev launchers.

For example, at my company, we have a "support@" email that is broadcasted to our operations and central support teams.

Reply:
We have support@devlaunchers.com.